packages/web/src/app/rental-business/page.tsx

```diff
@@ -0,0 +1,14 @@
+"use client";
+
+import React from "react";
+
+import { UseClientProvider } from "@sparcs-clubs/web/common/providers/UseClientProvider";
+import RentalMainFrame from "@sparcs-clubs/web/features/rental-business/frames/RentalMainFrame";
+
+const RentalBusiness = () => (
+  <UseClientProvider>
+    <RentalMainFrame />
+  </UseClientProvider>
```

Comment:
`UseClientProvider`는 `layout.tsx`으로 빼놔서 이제 페이지마다 넣어주지 않아도 됩니다!!
import에서도 제거해주면 좋을 것 같아용

```suggestion
  <RentalMainFrame />
```